.luacheckrc

```diff
@@ -0,0 +1,13 @@
+
+unused_args = false
```

Comment:
@kaeza Yes, digilines should definitely switch to `digiline.` or `digilines.` instead of `digiline:`, just like mesecons which has already switched over [some time ago](https://github.com/Jeija/minetest-mod-mesecons/commit/5be179bf110b44bdc06df6dbfde4e61487cf0635). There is no reason why we're using the colon `:` other than me not know what I was doing. If changing the name from `digiline` to `digilines` is preferred, I'd recommend to take care of switching to the period `.` notation at the same time so that mod developers don't need to update their dependent mods twice twice.

We could then provide wrappers for legacy mods and print out warnings every time a mod uses `digiline:` instead of `digilines.`.